environment/api_rds.tf

```diff
@@ -18,8 +18,8 @@ resource "aws_db_instance" "api" {
   license_model               = "postgresql-license"
   maintenance_window          = "sun:01:00-sun:02:30"
   monitoring_interval         = "0"
-  option_group_name           = terraform.workspace == "production02" ? "default:postgres-9-6" : "default:postgres-10"
-  parameter_group_name        = terraform.workspace == "production02" ? "default.postgres9.6" : "default.postgres10"
+  option_group_name           = contains(["production02", "training"], terraform.workspace) ? "default:postgres-9-6" : "default:postgres-10"
+  parameter_group_name        = contains(["production02", "training"], terraform.workspace) ? "default.postgres9.6" : "default.postgres10"
```

Comment:
If we remove these RDS will just pick the default

Reply:
Do you mean `option_group_name` and `parameter_group_name` @TomTucka ?

Reply:
Yep 🙂

Reply:
Nice - dropped now 👍